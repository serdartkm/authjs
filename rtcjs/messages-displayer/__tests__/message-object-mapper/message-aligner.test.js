import{h} from 'preact'
import MessageAligner from '../../lib/message-object-mapper/message-aligner'
import  render from 'preact-render-to-string';
describe("message-aligner component ", () => {

    it("renders correctly to the left", () => {
        const tree = render
           (<MessageAligner align="left">
                <div>Child component on the left side</div>
            </MessageAligner>)

        expect(tree).toMatchSnapshot();
    })
    it("renders correctly to the right", () => {
        const tree = render
            (<MessageAligner align="right">
                <div>Child component on the left side</div>
            </MessageAligner>)

        expect(tree).toMatchSnapshot();
    })
})



