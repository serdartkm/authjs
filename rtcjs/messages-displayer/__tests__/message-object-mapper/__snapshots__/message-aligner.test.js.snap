// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`message-aligner component  renders correctly to the left 1`] = `"<div style=\\"display: flex; justify-content: flex-end;\\"><div>Child component on the left side</div></div>"`;

exports[`message-aligner component  renders correctly to the right 1`] = `"<div style=\\"display: flex; justify-content: flex-end;\\"><div>Child component on the left side</div></div>"`;
